cst438_registration/src/main/java/com/cst438/service/GradebookServiceREST.java

```diff
@@ -43,7 +45,11 @@ public void enrollStudent(String student_email, String student_name, int course_
 	@Transactional
 	public void updateCourseGrades( @RequestBody FinalGradeDTO[] grades, @PathVariable("course_id") int course_id) {
 		System.out.println("Grades received "+grades.length);
-		
-		//TODO update grades in enrollment records with grades received from gradebook service
+
```

Comment:
This is well written, and easy to understand as well by following along. No comments are necessary.